src/ui/details/list_item.rs

```diff
@@ -8,7 +8,11 @@ use floem::{
 	reactive::{create_rw_signal, RwSignal, WriteSignal},
```

Comment:
This entire file is pure horror. I need to fix this all up. Can't sleep dreaming about this mess